src/client.lisp

```diff
@@ -88,7 +88,7 @@
                 (setf (gethash key **kwargs) val)
                 (process-**kwargs (rest (rest args))))))))
       (process-args args)
-      (setf (gethash "*args" **kwargs) *args)
+      (setf (gethash "*args" **kwargs) (nreverse *args))
```

Comment:
I'd prefer this to happen up in the `labels` definitions, but this is also OK.

Reply:
Ok, I'll take a look.

Reply:
Hmmm, might be misunderstanding your suggestion, but on second glance, I think reversing once here is preferable to `APPEND`ing or `NCONC`ing in the `LABELS` recursion above, since it's just mutating the closed-over `*ARGS`.  I could always hold on to the tail of the list and push there, but that seems more complicated.

Or did you mean modify the `LABELS` to pass an explicit accumulator and reverse that prior to handing it off to `process-**kwargs`?

Reply:
I think the latter.

Reply:
Done